src/css/controls/flags/cast-available.less

```diff
@@ -3,4 +3,13 @@
     .jw-icon-airplay {
         display: flex;
     }
+
+    &.jw-state-idle:not(.jw-flag-audio-player) {
```

Comment:
@hghazzi this has been moved to states.less under idle-state, and includes cardboard icon. you can delete this